src/Storages/ExternalStream/StorageExternalStreamImpl.h

```diff
@@ -46,7 +46,11 @@ class StorageExternalStreamImpl : public std::enable_shared_from_this<StorageExt
 
     FormatSettings getFormatSettings(const ContextPtr & context) const
     {
-        return settings->getFormatSettings(context);
+        auto ret = settings->getFormatSettings(context);
+        /// This is needed otherwise using an external stream with ProtobufSingle format as the target stream
+        /// of a MV (or in `INSERT ... SELECT ...`), i.e. more than one rows sent to the stream, exception will be thrown.
+        ret.protobuf.allow_multiple_rows_without_delimiter = true;
```

Comment:
What scenarios require reporting an error?

Reply:
I can't think of any scenarios reporting an error.